### ISSUE TITLE
False positive: Expression of type Enum cannot be assigned to member of class

### ISSUE THREAD

**Issue by harvastum** [microsoft/pyright#1110]
**Describe the bug**
Pyright incorrectly reports assigning an Enum value to a class member as an error.

**To Reproduce**
This snippet shows the problem:
```python
from enum import Enum


class Status(Enum):
    BAD = 0
    GOOD = 1


class Myclass():
    def __init__(self):
        self.status = Status.BAD


myobj = Myclass()
myobj.status = Status.GOOD  # this works perfectly well, yet Pyright has an issue with it.
```
**Expected behavior**
No warnings.

**VS Code extension or command-line**
Name: Pyright
Id: ms-pyright.pyright
Description: VS Code static type checking for Python
Version: 1.1.79
Publisher: ms-pyright
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=ms-pyright.pyright

**Comment by erictraut** [comment]
Thanks for reporting this bug. It will be fixed in the next release.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.81, which I just published. It will also be included in the next release of pylance.
